lib/xml/encrypt.js

```diff
@@ -29,8 +29,29 @@ exports.unencrypted = function (xml, callback) {
 exports.encrypted = function (encryptOptions) {
   return function encrypt(xml, callback) {
     xmlenc.encrypt(xml, encryptOptions, function (err, encrypted) {
-      if (err) return callback(err);
-      callback(null, utils.removeWhitespace(encrypted));
+      if (err) {
+        // Attempt to fix errors and retry
+        console.warn(
```

Comment:
@david-renaud-okta If I understand correctly we are adding a few of "console" calls, that haven't been used before.
I'm a bit worried about the consumers of this library and control they will have (or lack of it) to disable these logs, given that in some cases these PEMs might be coming from users, they might be spamming consumer systems with tons of warnings. Unless we provide some more sophisticated solution of signing for log events/injecting logger, maybe it makes sense to remove these logs?

Reply:
The idea behind the warning was to discourage using malformed PEMs but I can how that would be problematic when they come from users. It may be difficult to completely eliminate this kind of error which could create a lot of noise for something that doesn't impact the overall functionality.

I've removed the log